app/src/main/java/com/sun/moviedb/utils/BindingUtils.kt

```diff
@@ -10,15 +10,45 @@ import com.bumptech.glide.load.resource.bitmap.RoundedCorners
 import com.bumptech.glide.request.RequestOptions
 import com.sun.moviedb.MovieApplication
 import com.sun.moviedb.R
+import com.sun.moviedb.data.entity.Movie
+import com.sun.moviedb.view.adapter.MovieHomeAdapter
 
 /**
  * Created by nguyenxuanhoi on 2019-08-17.
  * @author nguyen.xuan.hoi@sun-asterisk.com
  */
 object BindingUtils {
+    @BindingAdapter("bindTextCategory")
+    @JvmStatic
+    fun textCategory(textView: TextView, query: String) {
+        when (query) {
+            CategoryQuery.UP_COMING -> textView.text = MovieApplication.applicationContext!!.getString(R.string.title_up_coming)
+            CategoryQuery.POPULAR -> textView.text = MovieApplication.applicationContext!!.getString(R.string.title_popular)
+            CategoryQuery.NOW_PLAYING -> textView.text = MovieApplication.applicationContext!!.getString(R.string.title_now_playing)
+            CategoryQuery.TOP_RATE -> textView.text = MovieApplication.applicationContext!!.getString(R.string.title_top_rate)
+        }
+    }
+
+    @BindingAdapter("android:src")
+    @JvmStatic
+    fun bindImageFromUrl(imageView: ImageView, imageUrl: String?) {
+        imageUrl?.let {
+            var requestOptions = RequestOptions()
+            requestOptions = requestOptions.transforms(CenterCrop(), RoundedCorners( MovieApplication.applicationContext!!.resources.getDimensionPixelSize(R.dimen.dp_8)))
+            Glide.with(imageView.context).applyDefaultRequestOptions(requestOptions).load(StringUtils.getImage(it)).into(imageView)
```

Comment:
1 dòng k quá 80 kí tự
chú ý format code
```suggestion
            Glide.with(imageView.context)
                    .applyDefaultRequestOptions(requestOptions)
                    .load(StringUtils.getImage(it))
                    .into(imageView)
```